docs/applications/mobu/configuring.rst

```diff
@@ -77,16 +77,20 @@ Here is a more complex example that runs a set of notebooks as a test:
          options:
            repo_url: "https://github.com/lsst-sqre/system-test.git"
            repo_branch: "prod"
+           exclude_dirs: ["some/experiment/dir", "another/dir"]
            max_executions: 1
          restart: true
 
 Here, note that the UID and primary GID for the user are specified, so this example will work in deployments that do not use Firestore and synthesized user private groups.
 
-This uses the business ``NotebookRunner`` instead, which checks out a Git repository and runs all notebooks at the top level of that repository.
+This uses the business ``NotebookRunner`` instead, which checks out a Git repository and runs all notebooks in that repository, at the top level and any directories and subdirectories.
+
 The repository URL and branch are configured in ``options``.
 ``options.max_executions: 1`` tells mobu to shut down and respawn the pod after each notebook.
 This exercises pod spawning more frequently, but does not test the lab's ability to run a long series of notebooks.
+Those directories are relative to the repo root.
```

Comment:
This sentence seems to be in the wrong spot. I think it was supposed to be after `exclude_dirs` below.